aws/tf/modules/sra/databricks_workspace/classic_cluster/cluster_configuration.tf

```diff
@@ -0,0 +1,37 @@
+// Terraform Documentation: https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/cluster
```

Comment:
Why comment this all out instead of put in a customization, or just never call the module right?
Seems odd to have a reference architecture have a fully commented out file